src/shared/toggle_panel.rs

```diff
@@ -5,116 +5,54 @@ live_design! {
     import makepad_widgets::theme_desktop_dark::*;
```

Comment:
Please ignore this file update.

I can't import this widget from makepad right now so I'm updating the version here in Moxin.
We should delete this file and use makepad's exported panel instead before merging.